python/GafferImageUI/OpenImageIOReaderUI.py

```diff
@@ -122,6 +122,16 @@
 
 		],
 
+		"__dataType" : [
```

Comment:
Since this is a public plug (intended for external use), it doesn't need the "__" prefix.